drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -622,8 +622,8 @@ static int stop_gps(struct device *dev, bool is_timeout)
 	retval = nrf_setsockopt(drv_data->socket,
 				NRF_SOL_GNSS,
 				NRF_SO_GNSS_STOP,
-				NULL,
-				0);
+				0,
```

Comment:
The fourth argument is a pointer, so NULL was a better fit there. Unless you wanted to provide the actual (uint32_t) value of 0, then you need to create a variable and provide its address.

Reply:
I agree, I will change the 0 to &delete_mask. 
The bsdlib side is done in parallel, and as late change we will add both checking that there is no NULL pointer, and update the documentation.